test/output/fixable.json

```diff
@@ -1,5 +1,5 @@
 {
-    "basePath": "/v1",
+    "basePath": "/v1/",
```

Comment:
Lets figure iojs issue. It'a outputting different basePath which is serious. A lot of people (including me) are using iojs.